configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -793,7 +793,76 @@
                 </p>
 
             </div>
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.TraumaticEvent"/>
+                    </label>
+                    <obs conceptId="PIH:12362" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
+                </p>
+
+            </div>
+        </div>
+
+        <div class="section-container" id="radio_button_container">
+
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.PoorIntrospection"/>
+                    </label>
+                    <obs conceptId="CIEL:166051" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
```

Comment:
@Dutervil I recommend that you reword the question OR answers.  

The question/label should be "Introspection" not "Poor introspection".  I think the answers should be "normal" and abnormal.

If you want the question to be "Poor introspection" with y/n, you can use that same concept and label with concepts of answers of "Poor" (yes label) and "normal" (no label)